Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/RCFCaster.java

```diff
@@ -254,11 +283,11 @@ public List<AnomalyDescriptor> processSequentially(double[][] data, Function<Ano
                     if (cacheDisabled) { // turn caching on temporarily
                         forest.setBoundingBoxCacheFraction(1.0);
                     }
-                    long timestamp = preprocessor.getInternalTimeStamp();
                     int length = preprocessor.getInputLength();
-                    for (double[] point : data) {
+                    for (int i = 0; i < data.length; i++) {
+                        double[] point = data[i];
```

Comment:
can maybe also add a new null check for point? reasonable to leave as is too though

Reply:
will add a null check